src/navbar/navbar-items.tsx

```diff
@@ -62,6 +62,25 @@ export const NavbarItems = <T,>({
         setShowSubMenu(false);
     };
 
+    const addAriaHiddenToIcon = (children: React.ReactNode) => {
```

Comment:
hmm given that the navbar items are totally customisable, I think the `aria-hidden` handling should be the responsibility of the consumer

also, for icons, as they indicate some functionality within a navbar, they shouldn't be hidden and should be labelled appropriately with `aria-label`